README.rst

```diff
@@ -32,48 +32,38 @@ following dependencies:
 
 - Python 2.7, 3.5+
 - NumPy
-- scikit-build >=0.8.1
-- cmake
+- scikit-build
```

Comment:
I noticed this: https://github.com/conda-forge/slycot-feedstock/blob/master/recipe/meta.yaml#L32, so you may need to leave a version specifier.

Reply:
Thanks, will do.

Reply:
As of cb791af513e4d38bd950b849acfb2a34e9a432b2 this is specified.